docs/architecture.md

```diff
@@ -27,7 +27,7 @@ CNI plugin. All Antrea Controller, Agent, OVS daemons, and `antrea-cni` bits
 are included in a single Docker image. Antrea also has a command-line tool
 called `antctl`, and an [Octant](https://github.com/vmware-tanzu/octant) UI plugin.
 
-<img src="/docs/assets/arch.svg.png" width="600" alt="Antrea Architecture Overview">
+![Antrea Architecture Overview](assets/arch.svg.png)
```

Comment:
@ccbayer @McCodeman is this necessary for netlify to render the image properly? Removing the fixed-width means that the images are way to big, at least when visualizing the markdown files in Github. 

Is there a syntax that works for both, e.g. `![Antrea Architecture Overview](assets/arch.svg.png){ width=600 }` ?

Reply:
@antoninbas there is CSS that will auto-scale the images so that they do not break the layout on the front-end.

Reply:
Sounds good. I am just pointing out that this should not be merged as is, because whoever looks at the documentation in Github will see this: https://github.com/vmware-tanzu/antrea/blob/50b5e85a2c7b6230f9535d5723b662f130d8acef/docs/architecture.md

Reply:
@antoninbas Looking into this again - there are two issues at hand:

* GitHub Markdown will only support using HTML width - there's no parameter or markdown flavor that supports passing through image params that will work for both the site AND for github markdown.
* the more important issue is the image path. by using markdown, we are able to keep it relative; by using HTML it interprets the path incorrectly on the site because it can't compile the "relative" paths from markdown into html.

Reply:
@antoninbas i was not able to figure this out in a clean way. suggestion is to just scale down the images to a better size that will work for both. I will address for these specific images.